client/src/main/java/io/confluent/kafka/schemaregistry/client/SchemaRegistryClientConfig.java

```diff
@@ -231,4 +234,10 @@ public static Map<String, Object> getClientSslConfig(Map<String, ?> configs) {
         .filter(e -> e.getKey().startsWith(SSL_PREFIX))
         .collect(Collectors.toMap(e -> e.getKey(), e -> e.getValue()));
   }
+
+  public static boolean getRandomizeStartingUrl(Map<String, ?> configs) {
+    return configs != null && configs.containsKey(RANDOMIZE_STARTING_URL)
+        ? (Boolean) configs.get(RANDOMIZE_STARTING_URL)
```

Comment:
You'll want to handle String as well. Look at how some of the other methods in this class handle String props